src/mlpack/methods/ann/gan_policies/standard_gan.hpp

```diff
@@ -0,0 +1,180 @@
+/**
+ * @file standard_gan.hpp
+ * @author Shikhar Jaiswal
+ *
+ * Implementation of the standard GAN policies.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_GAN_POLICIES_STANDARD_GAN_HPP
+#define MLPACK_METHODS_ANN_GAN_POLICIES_STANDARD_GAN_HPP
+
```

Comment:
Not sure, which one is is going to be picked up first, so perhaps we have to include ``gan.hpp``here.

Reply:
I don't think it fixes our issue I tried doing the above to no avail.